src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.html

```diff
@@ -32,6 +32,8 @@
   (optionSelected)="select($event.option.value)"
   autoActiveFirstOption
   [hideSingleSelectionIndicator]="true"
+  [class.custom-panel]="true"
+  [style.max-height.px]="maxPanelHeight"
```

Comment:
Have you tried finding a purely css property to get this working? Having layout calculations in the typescript code adds a lot of complexity

Reply:
Yes, I tried using only CSS. It works fine with just a few options (like 5–8). But when there are many options (like 10 or more), it shows two scrollbars. 
This also happens on small screens, even with fewer options. Because of that, I decided to handle it with typeScript code.